about.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sorting-Algorithms-in-Javascript</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/77ebb379f6.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../css/main.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar__container wrapper">
+            <a href="javascript:void(0)"> <img src="/img/logo.svg" alt="Website logo"> </a>
+            <div class="navbar__list-container">
+                <ul class="navbar__list">
+                    <li class="navbar__list-item"><a href="news.html" class="navbar__list-link">News</a></li>
+                    <li class="navbar__list-item"><a href="algorithm.html" class="navbar__list-link">Algorithm</a></li>
+                    <li class="navbar__list-item"><a href="about.html" class="navbar__list-link">About</a></li>
+                </ul>
+            </div>
+            <a href="#contact" class="navbar__btn">Contact us</a>
+        </div>
+    </nav>
+
+    <main>
+        <section class="about">
+            <div class="wrapper">
+                <div class="about__container">
+                    <div class="about__text">
+                        <h2 class="about__header">My name is Thomas...</h2>
+                        <p class="about__description">
+                            I'm a third year Applied Computer Science student at the AGH University of Science and
+                            Technology in Krakow. I have been learning Front-End technologies for a year now and this
+                            time was just enough to make sure that this is my place in the industry.</p>
+
+                        <p class="about__description">This website is one of my university projects, but during the
+                            process it turned out to be
+                            one of my lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed id urna sed tortor
+                            dignissim semper. Nulla nec dapibus urna, a ultrices ante. Etiam purus lorem, placerat vitae
+                            hendrerit id, ultrices sed est. Phasellus scelerisque volutpat consectetur. Lorem ipsum
+                            dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                    <div class="about__image">
```

Comment:
Sugestia: użyj image placeholdera, np. https://dummyimage.com/

Reply:
Dzięki za radę. Wstawiłem image placeholder, ale niestety nie do końca jestem zadowolony, ponieważ mimo tego, że na stronie zmieniałem tekst to u mnie cały czas pokazywało wymiary obrazka zamiast tekstu.